base/_breakpoints.scss

```diff
@@ -14,6 +14,7 @@ $breakpoint-mobile-only: "#{$screen} and (max-width:#{$breakpoint-tablet - 1})";
 $breakpoint-tablet-up: "#{$screen} and (min-width:#{$breakpoint-tablet})";
 $breakpoint-tablet-only: "#{$screen} and (min-width:#{$breakpoint-tablet}) and (max-width:#{$breakpoint-desktop - 1})";
 
+$breakpoint-desktop-down: "#{$screen} and (max-width:#{$breakpoint-desktop - 1})";
```

Comment:
According to our naming convention for breakpoints `breakpoint-desktop-down` suggest that this breakpoint includes desktop devices. What do you think about `$breakpoint-tablet-down`?